std/assembly/bindings/dom.ts

```diff
@@ -115,6 +115,23 @@ export declare namespace String {
   export function fromCodePoints(codepoints: i32[]): externref;
 }
 
+export declare namespace Object {
```

Comment:
These bindings only get us so far. We can't do anything with `externref` results directly in AssemblyScript code. For example, we can't write

```js
const el = document.createElement('div')
el.setAttribute('foo', 'bar')
document.body.append(el)
```

For this reason, we can't simply say that the loader API is obsolete. Loader API is needed in order to make realistic portable code that compiles to JS (without bindings) or WebAssembly (with bindings).